meta-refkit-industrial/recipes-image/images/refkit-image-industrial.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "PROJECTNAME image for Industrial Robotics profile."
```

Comment:
Out of curiosity, is 'PROJECTNAME' supposed to be expanded/replaced at some point?